dspace-api/src/test/resources/test-config.properties

```diff
@@ -13,3 +13,5 @@ test.folder = ./target/testing/
 test.bitstream = ./target/testing/dspace/assetstore/ConstitutionofIreland.pdf
 test.exportcsv = ./target/testing/dspace/assetstore/test.csv
 test.importcsv = ./target/testing/dspace/assetstore/testImport.csv
+test.scopus = ./target/testing/dspace/assetstore/scopus-ex.xml
+test.scopus-empty = ./target/testing/dspace/assetstore/scopus-empty-resp.xml
```

Comment:
These two changes can be removed as you moved these files over to `src/test/resources/org/dspace/app/rest`